test/e2e/scenarios/bare-metal-testdriver.go

```diff
@@ -91,11 +100,15 @@ func (d *baremetalDriver) SkipUnsupportedTest(pattern testpatterns.TestPattern)
 
 		// We have volume and exec pvc test for default fs (equals to xfs) in short CI
 		// Not need to perform them for other filesystems
-		if pattern.FsType == "xfs" || pattern.FsType == "ext4" || pattern.FsType == "ext3" {
+		if pattern.FsType == xfsFs || pattern.FsType == ext4Fs || pattern.FsType == ext3Fs {
 			e2eskipper.Skipf("Should skip tests in short CI suite -- skipping")
 		}
 	}
 
+	if pattern.AllowExpansion && pattern.VolMode == corev1.PersistentVolumeBlock {
```

Comment:
Is it works for short-ci suite without skipping expansion tests?

Reply:
AFAIK short-ci isn't use community tests